bitnami/airflow/templates/triggerer/statefulset.yaml

```diff
@@ -127,12 +127,24 @@ spec:
             {{- if .Values.triggerer.extraEnvVars }}
             {{- include "common.tplvalues.render" (dict "value" .Values.triggerer.extraEnvVars "context" $) | nindent 12 }}
             {{- end }}
-          {{- if or .Values.triggerer.extraEnvVarsCM .Values.triggerer.extraEnvVarsSecret .Values.extraEnvVarsCM .Values.extraEnvVarsSecret .Values.extraEnvVarsSecrets }}
```

Comment:
Note to self: Check if this conditional is missing triggerer.extraEnvVarsSecrets incidentally